dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -59,7 +59,9 @@ def next
       redirect_to @script.finish_url
       return
     end
-    redirect_to(build_script_level_path(next_script_level)) && return
+    path = build_script_level_path(next_script_level)
+    path += "?section_id=#{params[:section_id]}" if params[:section_id]
```

Comment:
I would wonder if there's a more railsy way to do this rather than just appending `?foo`. I would have thought os, but my two minutes of googling didn't find anything so this might be fine.